### PR TITLE
Flexiti logo

### PR DESCRIPTION
**Pull request by jjsegovia**
** Checklist **
- [x ] All icons have a corresponding entry in `db/payment_icons.yml`
- [ x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [ x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [ x] I am confident that all icons are clear and easy to read/understand
- [ x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Review by larouxn — APPROVED** [review]
LGTM 🎩 
Screen Shot 2021-11-09 at 17 01 48

File: app/assets/images/payment_icons/flexiti.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... 2-2h32"/><path d="M37.5 21.5c0 1.1-.9 2-2 2h-33c-1.1 0-2-.9-2-2V2.4c0-1.1.9-2 2-2h32.9c1.1 0 2 .9 2 2v19.1z" opacity=".07" fill="none" stroke="#010101" stroke-miterlimit="10"/><path d="M21.1 11.1l-.8-1.1H18l2.2 3.2.9-2.1z" fill="#e94779"/><path d="M8.2 16.8V6.9h1.9v9.9H8.2zM31.3 14.8c0 .3.1.4.5.4h.3v1.6h-1.5c-.9 0-1.3-.4-1.2-1.3v-4h-.8V10h.8V8.1h1.9V10h.9v1.6h-.9v3.2zM35 8c0 .6-.4 1-1 1s-1.1-.4-1.1-1 .4-1.1 1.1-1.1c.6 0 1 .5 1 1.1zm-.1 8.8H33V10h1.9v6.8zM7.2 8.7V6.9h-.7c-1.9 0-2.8.8-2.8 2.7v.4H3v1.6h.8v5.3h1.9v-5.3h1.6V10H5.6v-.4c0-.5.2-.8.6-.9h1zM12.9 13.9c.2 1 .9 1.5 1.8 1.5.7 0 1.3-.3 1.6-.9l1.7.5c-.7 1.3-1.9 2.1-3.3 2.1-2.2 0-3.7-1.5-3.7-3.7s1.5-3.7 3.6-3.7c2.2 0 3.7 1.5 3.7 3.8v.4h-5.4zm3.4-1.1c-.2-.9-.8-1.5-1.8-1.5-.9 0-1.5.5-1.7 1.5h3.5z" fill="#31393e"/><path d="M23.1 12.3c.3-.7.9-1.5 1.5-1.8.3-.1.5-.2.9-.2-.6-.3-1.1-.6-1.6-.6h-.2c-.4 0-.8.1-1.1.2-1.1.7-1.9 3.3-2.5 4.5-.3.7-.9 1.5-1.5 1.8-.3.1-.5.2-.9.2.6.3 1.1.6 1.6.6h.2c.4 0 .8-.1 1.1-.2 1-.6 1.9-3.3 2.5-4.5z" fill="#e94779"/><path d="M27.9 8c0 .6-.4 1-1 1s-1.1-.4-1.1-1 .4-1.1 1.1-1.1c.6 0 1 .5 1 1.1zm-.1 8.8h-1.9V10h1.9v6.8z" fill="#31393e"/><path d="M22 15.7l.8 1.2H25l-2.2-3.2c-.2.6-.5 1.3-.8 2z" fill="#e94779"/></svg>
```
[the remark this answers is not in the document]
Comment (jjsegovia) [reply app/assets/images/payment_icons/flexiti.svg]: no worries, it's done

**Review by tauthomas01 — APPROVED** [review]

**Comment by larouxn** [comment]
👋 For clarity, the icon won't be usable across Shopify yet as we need to roll it out. Hope to have it live within the month but Black Friday/Cyber Monday might delay it a bit.

**Comment by larouxn** [comment]
👋 Aiming to release a new version of this gem soon. Hoping to have this new icon live by end of next week at the latest.

**Comment by larouxn** [comment]
👋 @jjsegovia, your icon should be available everywhere now. Have a great day! ☀️

**Pull request merged**